contracts/base/Dispatcher.sol

```diff
@@ -330,7 +330,8 @@ abstract contract Dispatcher is Payments, V2SwapRouter, V3SwapRouter, Callbacks,
             // 0x20 <= command
         } else {
             if (command == Commands.EXECUTE_SUB_PLAN) {
-                (bytes memory _commands, bytes[] memory _inputs) = abi.decode(inputs, (bytes, bytes[]));
+                bytes calldata _commands = inputs.toBytes(0);
```

Comment:
entirely unrelated to the PR but just seeing all the custom calldataloads above for the first time.. sucks we dont have calldata version of abi.decode.. Wonder if there's a nice library or something we can write/use to save from accidentally fatfingering offsets or something

Reply:
ooooh maybe soon tm? https://github.com/ethereum/solidity/issues/13518

Reply:
haha yeah the PR that added these decodes was made by the person who made that solidity issue :)

Reply:
fingers crossed it happens soon!